graphics/tinygl/opinfo.h

```diff
@@ -84,6 +84,8 @@ ADD_OP(DepthFunc, 1, "%d")
 ADD_OP(StencilFunc, 3, "%C %d %d")
 ADD_OP(StencilOp, 3, "%C %C %C")
 
+ADD_OP(PolygonStipple, 1, "%d")
```

Comment:
Op count is incorrect, should probably be `ADD_OP(PolygonStipple, 128, "")`

Reply:
Fixed!

Reply:
It looks like this was not fixed properly, the op count for `PolygonOffset` was changed instead, now they are both wrong. :(

Reply:
Fixed